django-rgd/rgd/utility.py

```diff
@@ -177,20 +75,54 @@ def get_or_create_no_commit(model, defaults=None, **kwargs):
         return model(**defaults), True
 
 
-@contextmanager
-def url_file_to_local_path(
-    url: str, num_blocks=128, block_size=128, override_name=None
-) -> Generator[Path, None, None]:
-    with safe_urlopen(url) as remote:
-        if override_name:
-            suffix = override_name
-        else:
-            suffix = PurePath(os.path.basename(url)).name
-        with tempfile.NamedTemporaryFile('wb', suffix=suffix) as dest_stream:
+def _get_s3_client():
+    if boto3.session.Session().get_credentials():
+        s3 = boto3.client('s3')
+    else:
+        # No credentials present (often the case in dev), use unsigned requests
+        s3 = boto3.client('s3', config=Config(signature_version=UNSIGNED))
+    return s3
```

Comment:
We may want to double-check this... I'm not positive if this is the correct way to check for valid credentials.

Reply:
Update: this does not seem to work when credentials are specified in the environment as `AWS_ACCESS_KEY_ID`/`AWS_SECRET_ACCESS_KEY`